### DISCUSSION TITLE
Подключение к RTC в Discord

### DISCUSSION THREAD

**Discussion by cuneal** [remittor/zapret-openwrt#8] [Ideas]
Есть ли возможность так же обойти спотифай и инсту , к примеру?

**Comment by bombsnuccer** [comment]
мне помогло всё удалить, через web интерфейс, а также на всякий случай удалил папку zapret из opt
Установил всё заново по инструкции
Остановил Service daemons control
Переместил 50-discord из /opt/zapret/init.d/openwrt/custom.d.examples в /opt/zapret/init.d/openwrt/custom.d
Запустил Service daemons control
Всё заработало
